integration

```diff
@@ -65,6 +65,7 @@
   "alryaz/hass-turkov",
   "altfoxie/ha-sberdevices",
   "Altrec/remko_mqtt-ha",
+  "alves-dev/stackspot-homeassistant",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>